optimize/self-managed/optimize-deployment/configuration/history-cleanup.md

```diff
@@ -112,8 +106,10 @@ historyCleanup:
     enabled: true
 ```
 
+<!-- This section needs attention -->
+
 :::note
-The ingested event cleanup does not cascade down to potentially existing [event-based processes](components/userguide/additional-features/event-based-processes.md) that may contain data originating from ingested events. To make sure data of ingested events is also removed from event-based processes, you need to enable the [Process Data Cleanup](#process-data-cleanup) as well.
+The ingested event cleanup does not cascade down to potentially existing [event-based processes](#) that may contain data originating from ingested events. To make sure data of ingested events is also removed from event-based processes, you need to enable the [Process Data Cleanup](#process-data-cleanup) as well.
 :::
 
 </TabItem>
```

Comment:
With the above two tabs removed, we can also greatly simplify the example below to just be:

```
historyCleanup:
  cronTrigger: '0 1 * * 0'
  ttl: 'P1Y'
  processDataCleanup:
    enabled: true
    cleanupMode: 'variables'
    perProcessDefinitionConfig:
      'VeryConfidentProcess':
        ttl: 'P1M'
        cleanupMode: 'all'
      'KeepTwoMonthsProcess':
        ttl: 'P2M'
```

The explanation below can be simplified to correspond to this change too